test/EntityFramework6.Npgsql.Tests/Support/EntityFrameworkTestBase.cs

```diff
@@ -213,6 +223,85 @@ private static DbCompiledModel CreateModel(NpgsqlConnection connection)
+            var rowType = RowType.Create(
+                modelBlogStoreType.Properties.Select(x =>
+                {
+                    var clone = EdmProperty.Create(x.Name, x.TypeUsage);
+                    clone.CollectionKind = x.CollectionKind;
+                    clone.ConcurrencyMode = x.ConcurrencyMode;
+                    clone.IsFixedLength = x.IsFixedLength;
+                    clone.IsMaxLength = x.IsMaxLength;
+                    clone.IsUnicode = x.IsUnicode;
+                    clone.MaxLength = x.MaxLength;
+                    clone.Precision = x.Precision;
+                    clone.Scale = x.Scale;
+                    clone.StoreGeneratedPattern = x.StoreGeneratedPattern;
+                    clone.SetMetadataProperties(x
+                        .MetadataProperties
+                        .Where(metadataProerty => !clone
+                            .MetadataProperties
+                            .Any(cloneMetadataProperty => cloneMetadataProperty.Name.Equals(metadataProerty.Name))));
+                    return clone;
+                }),
+                null);
+
+            var getBlogsFunc = EdmFunction.Create(
+                "StoredGetBlogsFunction",
+                "BloggingContext",
+                DataSpace.SSpace,
+                new EdmFunctionPayload
+                {
+                    ParameterTypeSemantics = ParameterTypeSemantics.AllowImplicitConversion,
+                    Schema = "dbo",
+                    IsComposable = true,
+                    IsNiladic = false,
+                    IsBuiltIn = false,
+                    IsAggregate = false,
+                    StoreFunctionName = "GetBlogsByName",
+                    ReturnParameters = new[]
+                    {
+                            FunctionParameter.Create("ReturnType1", rowType.GetCollectionType(), ParameterMode.ReturnValue)
+                    },
+                    Parameters = new[]
+                    {
+                            FunctionParameter.Create("Name", dbModel.ProviderManifest.GetStoreTypes().First(x=>x.ClrEquivalentType == typeof(String)), ParameterMode.In)
+                    }
+                },
+                null);
+            dbModel.StoreModel.AddItem(getBlogsFunc);
+
+            var modelBlogConceptualType = dbModel.ConceptualModel.EntityTypes.First(x => x.Name == typeof(Blog).Name) as EntityType;
+            EdmFunction getBlogsFuncModel = EdmFunction.Create(
+                "GetBlogsByName",
+                dbModel.ConceptualModel.Container.Name,
+                DataSpace.CSpace,
+                new EdmFunctionPayload
+                {
+                    IsFunctionImport = true,
+                    IsComposable = true,
+                    Parameters = new[] 
+                    {
+                        FunctionParameter.Create("Name", PrimitiveType.GetEdmPrimitiveTypes().First(x=>x.ClrEquivalentType == typeof(string)), ParameterMode.In)
```

Comment:
Nitpick: spacing around "=>" in ```x=>x.ClrEquivalentType```. Also, the line is too long and it is not readable.
Perhaps pull all edm types into local variables above your function declarations and move it into a separate function to keep all the code for setting up this function together.